src/common/components/elements/Dropdown.jsx

```diff
@@ -0,0 +1,42 @@
+import { Fragment } from 'react'
+import { Menu, Transition } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/solid';
+import { LinkedItem } from "./LinkedItem";
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ')
+}
+
+export default function Example({head,name,href,className}) {
```

Comment:
we already have a package called `clsx`. Try using that instead of `classNames` function.
and accept options as props and map through it for dropdown menu